PWGCF/FemtoDream/FemtoDreamParticleHisto.h

```diff
@@ -223,28 +226,27 @@ class FemtoDreamParticleHisto
   }
 
   template <o2::aod::femtodreamMCparticle::MCType mc, typename T>
-  void fillQA_debug(T const& part, int MomemtumType)
+  void fillQA_debug(T const& part, int MomentumType)
   {
     float momentum;
-    switch (MomemtumType) {
-      case 0:
+    switch (MomentumType) {
+      case aod::femtodreamparticle::kPt:
         momentum = part.pt();
         break;
-      case 1:
+      case aod::femtodreamparticle::kPreco:
         momentum = part.p();
         break;
-      case 2:
-        LOG(warn) << "Mometum at TPC inner wall is only available if the Producer Task generates debug information.";
+      case aod::femtodreamparticle::kPtpc:
+        LOG(warn) << "Momentum at TPC inner wall is only available if the Producer Task generates debug information.";
         momentum = part.tpcInnerParam();
         break;
       default:
-        LOG(warn) << "MomentumType " << momentum << " no implemented. Use pT of the Track for debug Histograms.";
+        LOG(warn) << "MomentumType " << MomentumType << " no implemented. Use pT of the Track for debug Histograms.";
         momentum = part.pt();
     }
 
```

Comment:
The idea of having an `enum` type is that you instead of passing an`int` you pass a variable that has the `enum` type (I'm surprised the compiler allowed you to hide the `enum` type  using the same name for a variable) in that way you have, at design level, full control of what the allowed values.
I approve it for the time being but change it in the next iteration

As it is now it is not only misleading but also dangerous in the medium term